### PR TITLE
fix not initialized value - reported by klocwork

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - fix not initialized value - IE IR parser

### CODE REVIEW

File: inference-engine/src/readers/ir_reader/ie_ir_parser.cpp [hunk 1]
```diff
@@ -82,11 +82,14 @@ bool getParameters(const pugi::xml_node& node, const std::string& name, std::vec
 }
 
 template <class T>
-bool stringToType(const std::string& valStr, T& value) {
+std::pair<T, bool> stringToType(const std::string& valStr) {
```
Comment (jdanieck) [inline inference-engine/src/readers/ir_reader/ie_ir_parser.cpp:85]: Seems that this bool return value is not used at all, why not to remove it entirely?

**Review by jdanieck — APPROVED** [review]

File: inference-engine/src/readers/ir_reader/ie_ir_parser.cpp (hunk 1 above)
Comment (pelszkow) [reply inference-engine/src/readers/ir_reader/ie_ir_parser.cpp]: Boolean value removed.

**Comment by jdanieck** [comment]
@lazarevevgeny I've heard that @ilyachur is OOO so I've ask you for review, we don't have permissions to merge. It would be also great if you could point whom we should ask for review during @ilyachur absence.

**Comment by jdanieck** [comment]
@tomdol do you have permissions to merge ? :)

**Review by GlebKazantaev — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
@GlebKazantaev @lazarevevgeny Please merge these changes.

**Review by tomdol — APPROVED** [review]

**Pull request merged**